### PR TITLE
MG-239: Add correct tessdata file

### PR DESCRIPTION
**Pull request by domenicrosati**

### CODE REVIEW

**Comment by sallyhill** [comment]
does the faster tessdata affect quality?

**Comment by domenicrosati** [comment]
tbd
Edit:
I am guessing that it is better though because it is thr data for tess4 rather than tess3

File: Dockerfile [hunk 1]
```diff
@@ -58,7 +58,7 @@ RUN cd / \
  && ldconfig \
  && cd .. \
  && rm -rf tesseract-*/ \
- && curl -sSL 'https://github.com/tesseract-ocr/tessdata/blob/master/eng.traineddata?raw=true' \
```
Comment (npars) [inline Dockerfile:61]: We should pull a fixed hash

**Comment by domenicrosati** [comment]
screenshot 2017-12-05 14 31 01

**Comment by domenicrosati** [comment]
Demo'd to @sallyhill / see the comment

**Review by npars — APPROVED** [review]

**Pull request merged**